graphql/connection_test.go

```diff
@@ -292,6 +305,107 @@ func TestConnection(t *testing.T) {
 			}
 		}
 	}`)
+
+	snap.SnapshotQuery("Pagination, sorts", `{
+		inner {
+			numbersAsc: innerConnectionWithSort(sortBy: "numbers", sortOrder: "asc", first: 5, after: "") {
+				totalCount
+				edges {
+					node {
+						id
```

Comment:
Same thing about being confusing to read the snapshots, can we add the fields we're sorting by here.

Reply:
For the snapshot? I've named the parent node for what we're sorting. Unfortunately, since these are virtual values, they're not super deuggable. If we were sorting actual fields that are available for querying, it would be easier. I think this is another argument to not add additional virtual fields here, and opt instead for a shorthand to existing sub-fields.

Reply:
hrm, maybe we can add a "dummy" node field that is the value of the sortable/filterable field (which would be the exact same as the field we're sorting/filtering by) just to make reading the snapshots easier.

Reply:
I've added that